<?xml version="1.0"?>
<launch>
  <arg name="scan_topic" default="/robot1/scan"/>
  <arg name="cmd_vel_topic" default="/robot1/cmd_vel"/>
  <arg name="odom_topic" default="/robot1/odom"/>
 

  <arg name="base_global_planner" default="srv_client_plugin/SrvClientPlugin"/>
  
  <arg name="base_local_planner" default="dwa_local_planner/DWAPlannerROS"/>


  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find uvg)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find uvg)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find uvg)/config/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find uvg)/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find uvg)/config/dwa_local_planner_params.yaml" command="load" />

    <rosparam file="$(find uvg)/config/move_base_params.yaml" command="load" />
    <rosparam file="$(find uvg)/config/global_planner_params.yaml" command="load" />
    <rosparam file="$(find uvg)/config/navfn_global_planner_params.yaml" command="load" />
    <param name="base_global_planner" value="$(arg base_global_planner)"/>
    <param name="base_local_planner" value="$(arg base_local_planner)"/>  
  </node>
</launch>